pkgs/applications/office/qownnotes/default.nix

```
@@ -47,8 +47,11 @@ stdenv.mkDerivation {
     botan2
   ] ++ lib.optionals stdenv.isLinux [ qtwayland ];
 
-  qmakeFlags = [
-    "USE_SYSTEM_BOTAN=1"
+  cmakeFlags = [
+    "-DQON_QT6_BUILD=ON"
+    "-DUSE_SYSTEM_BOTAN=1"
+    "-DBUILD_WITH_SYSTEM_BOTAN=ON"
+    "-DQT_TRANSLATIONS_DIR=./translations"
```

Maybe?
```suggestion
    "-DQT_TRANSLATIONS_DIR=${placeholder "out"}/share/qt/translations"
```
The relative path is relative to what?